conf/iCubFindDependencies.cmake

```diff
@@ -64,7 +64,7 @@ checkandset_dependency(IPOPT)
 checkandset_dependency(OpenCV)
 checkandset_dependency(Qt5)
 
-set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.39.1)
+set(MINIMUM_REQUIRED_icub_firmware_shared_VERSION 1.41.0)
```

Comment:
Hi @ale-git 

This has to be updated to `1.40.1` as per https://github.com/robotology/icub-firmware-shared/pull/99?notification_referrer_id=NT_kwDOADkJ1rMxMjk4OTAxMTk4MzozNzM4MDcw#discussion_r1846898167.

cc @marcoaccame

Reply:
Same as in https://github.com/robotology/icub-main/pull/989#discussion_r1846911095.